apps/web/src/components/molecules/PortableText.tsx

```diff
@@ -1,26 +1,21 @@
 /*eslint-disable*/
```

Comment:
Should probably not be here

Reply:
It is supposed to be there since there is a "any" tag in Portable text